client/transactions/list/index.js

```diff
@@ -203,7 +203,10 @@ export const TransactionsList = ( props ) => {
 		// Map transaction into table row.
 		const data = {
 			// eslint-disable-next-line camelcase
-			transaction_id: { value: txn.transaction_id },
+			transaction_id: {
+				value: txn.transaction_id,
+				display: txn.transaction_id,
```

Comment:
Id is displayed as text rather than clickable cell, need to wrap it like this:
```suggestion
				display: clickable( txn.transaction_id ),
```

Reply:
Fixed in commit: https://github.com/Automattic/woocommerce-payments/pull/1893/commits/0c5eea52d4549cfc1334ebed6860d334efb1828e